requirements.txt

```diff
@@ -68,4 +68,4 @@ pydoc-markdown~=3.13.0; python_version >= '3.5'             # Used for generatin
 sh~=1.14.2                                                  # Convenience library for running subprocesses in Python
 apispec==5.1.1                                              # Generate OpenApi definition out of pydoc comments
 apispec-webframeworks==0.5.2                                # Integration of apispec in Flask
-pymongo==4.0.1                                              # pymongo (metadata plugin)
\ No newline at end of file
+pymongo==4.0.1                                              # pymongo (metadata plugin)
```

Comment:
This should go into the "All dependencies needed in extras for rucio server/daemons should be defined here" section (Line 41) since it is an extra. 
Also you need to add it here: https://github.com/rucio/rucio/blob/81394b76f9d83b0f06ec6c0116fd0516418ea437/setuputil.py#L102 as an extra. So when you do `pip install rucio[pymongo]` it will install Rucio with that extra.

We probably have to add it to the containers as well... But I wonder if we should do that as a default or people must do it manually.